source/install/desktop-app-changelog.rst

```diff
@@ -228,6 +228,7 @@ Compatibility
 ~~~~~~~~~~~~~~~
 
 - Desktop App is supported on any supported Extended Support Release or a newer Mattermost server version.
+- Desktop App v5.2 is incompatible with server versions v9.1 and later.
```

Comment:
Seems that `Desktop App is supported on any supported Extended Support Release` will stop being true after the next 9.x one so wondering if we should also include 
```suggestion
- Desktop App is supported on any supported Extended Support Release up to v8.1 ESR.
- Desktop App v5.2 is incompatible with server versions v9.1 and later.
```